README.md

```diff
@@ -247,10 +252,68 @@ If you are running in an "air-gapped" environment, you may need to download the
 
 Once the migration script starts, you will be presented with some questions to specify the Minio instance, the credentials, and the Minio bucket name to migrate your package artifacts to. The script will attempt to automatically detect all of these from the running service, so you can usually just accept the defaults. Please refer to your `bldr.env` file if you need to explicitly type in any values.
 
-The migration script may take a while to move over the artifacts into Minio. During the script migration, the Depot services will continue to run as normal, however packages will not be downloadable until the artifacts are migrated over to Minio.  
+The migration script may take a while to move over the artifacts into Minio. During the script migration, the Depot services will continue to run as normal, however packages will not be downloadable until the artifacts are migrated over to Minio.
 
 Once the migration is complete, you will be presented with an option to remove the files in your `hab/svc/builder-api/data/pkgs` directory. You may want to preserve the files until you have verified that all operations are completing successfully.
 
+## Merging Database Shards
+
+This section is for installations of On-Premise Depot that were done *prior* to
+August 17th 2018. If you re-install or upgrade to a newer version of the
+On-Premise Depot, you will be required to also merge your database shards into
+the `public` Postgres database schema. Please follow the steps below.
+
+### Pre-requisites
+1. A copy of [merge-shards.sh](https://github.com/habitat-sh/builder/blob/master/tools/merge-shards/merge-shards.sh)
```

Comment:
For convenience (and air-gap scenarios) we should add a copy of that script to this repo..